user_modules/shared_testcases/testCasesForExternalUCS.lua

```diff
@@ -0,0 +1,179 @@
+---------------------------------------------------------------------------------------------
+-- Common functions for External_UCS test scripts
+---------------------------------------------------------------------------------------------
+local mobile_session = require("mobile_session")
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local json = require("modules/json")
+local sdl = require('SDL')
+
+local utils = { }
+
+-- [[ Variables ]]
+	utils.HMIAppIds = { }
+	utils.pts = nil
+
+-- [[ Functions ]]
+	function utils.createTableFromJsonFile(file)
+	  local f = io.open(file, "r")
+	  local content = f:read("*all")
+	  f:close()
+	  return json.decode(content)
+	end
+
+	function utils.createJsonFileFromTable(table, file)
+	  local f = io.open(file, "w")
+	  f:write(json.encode(table))
+	  f:close()
+	end
+
+	function utils.checkSDLStatus(test, expStatus)
+	  local actStatus = sdl:CheckStatusSDL()
+	  print("SDL status: " .. tostring(actStatus))
+	  if actStatus ~= expStatus then
+	    local msg = "Expected SDL status: " .. expStatus .. ", actual: " .. actStatus
+	    test:FailTestCase(msg)
+	  end
+	end
+
+	function utils.removeLPT()
```

Comment:
@istoimenova, The idea to clean all storage folder in order to simulate initial Local Policy Table creation